backend/app/use_case/recommendation.py

```diff
@@ -10,40 +10,91 @@
 openai.api_base = "https://api.openai.iniad.org/api/v1"
 
 
+def extract_json(openai_response: str) -> str:
+    """
+    Extract json from openai response
+    """
+    json_start_pos = openai_response.find("{")
+    json_end_pos = openai_response.rfind("}")
+    json = openai_response[json_start_pos : json_end_pos + 1]
+    return json
+
+
 def generate_recommendation_structured_format_query(query: StructuredQuery) -> str:
     place = query.place
     duration = query.duration
+    people_num = query.people_num
     budget = query.budget
     trip_style = query.trip_style
     interests = query.interests
-    foods = query.foods
+    trip_type = query.trip_type
+    remarks = query.remarks
```

Comment:
Sorry,my format was wrong. `remarks` is not needed for now so can you remove this and related one too?